21torr/hosting/3.2/config/packages/hosting.yaml

```diff
@@ -0,0 +1,4 @@
+hosting:
```

Comment:
Unrealted to current PR, but while your bundle is young, I would suggest to set proper namespacing, for example

```suggestion
torr_hosting:
```

or

```suggestion
21torr_hosting:
```

Reply:
Thanks for the suggestion, I will think about that for v4.

Reply:
I just did the change for v4 of the bundle.